benchmarks/module/core/bit_ceil/regular/bit_ceil.hpp

```diff
@@ -13,12 +13,9 @@ int main()
   auto lmin = eve::valmin(eve::as<EVE_VALUE>());
   auto lmax = eve::valmax(eve::as<EVE_VALUE>());
 
-  auto const std__bit_ceil = [](EVE_VALUE x) { return x > 0 ? x : EVE_VALUE(-x); };
```

Comment:
why not using std::bit_ceil when possible ?

Reply:
done